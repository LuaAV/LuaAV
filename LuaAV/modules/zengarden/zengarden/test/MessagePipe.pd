#N canvas 414 303 576 325 10;
#X obj 255 12 loadbang;
#X obj 228 13 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X msg 431 216 hello;
#X msg 507 147 2000;
#X obj 449 178 pipe 1000;
#X obj 431 261 print piped;
#X obj 392 93 t b b b;
#X obj 316 93 t b b b;
#X obj 208 93 t b b b;
#X obj 342 150 pipe 4000;
#X obj 343 261 print flushed;
#X msg 343 208 water;
#X obj 234 150 pipe 4000;
#X obj 235 261 print cleared;
#X msg 235 208 fire;
#X msg 289 125 flush;
#X msg 181 125 clear;
#X connect 0 0 6 0;
#X connect 0 0 7 0;
#X connect 0 0 8 0;
#X connect 1 0 6 0;
#X connect 1 0 7 0;
#X connect 1 0 8 0;
#X connect 2 0 5 0;
#X connect 3 0 4 1;
#X connect 4 0 2 0;
#X connect 6 0 4 0;
#X connect 6 1 3 0;
#X connect 6 2 2 0;
#X connect 6 2 4 0;
#X connect 7 0 15 0;
#X connect 7 1 9 0;
#X connect 7 2 9 0;
#X connect 8 0 16 0;
#X connect 8 1 12 0;
#X connect 8 2 12 0;
#X connect 9 0 11 0;
#X connect 11 0 10 0;
#X connect 12 0 14 0;
#X connect 14 0 13 0;
#X connect 15 0 9 0;
#X connect 16 0 12 0;
