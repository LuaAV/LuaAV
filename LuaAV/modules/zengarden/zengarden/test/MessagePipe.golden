[@ 0.000ms] piped: hello
[@ 0.000ms] flushed: water
[@ 0.000ms] flushed: water
[@ 1000.000ms] piped: hello
[@ 2000.000ms] piped: hello